### PR TITLE
bin/etc: make raft cf configurable

### PR DESCRIPTION
**Pull request by zhangjinpeng1987**
Make column family raft configurable.

### CODE REVIEW

**Comment by zhangjinpeng1987** [comment]
@ngaut @siddontang @BusyJay @disksing PTAL

**Comment by siddontang** [comment]
Any benchmark?

File: etc/config-template.toml [hunk 1]
```diff
@@ -128,6 +128,28 @@ enable-statistics = false
 # Same as rocksdb's default value (10 min).
 stats-dump-period-sec = 600
 
+# options for Column Family write
+# Column Family write used to store commit informations in MVCC model
+[rocksdb.writecf]
+compression_per_level = "lz4:lz4:lz4:lz4:lz4:lz4:lz4"
```
Comment (siddontang) [inline etc/config-template.toml]: s/_/-/

**Comment by BusyJay** [comment]
LGTM

**Comment by siddontang** [comment]
LGTM

**Pull request merged**